src/api/clients/IOClients/infra/VBase.ts

```diff
@@ -21,10 +22,30 @@ export class VBase extends InfraClient {
     })
   }
 
+  public resolveConflict = (bucketKey: string, path: string, content: any) => {
+    const data = [
+      {
+        op: 'replace',
+        path,
+        value: content,
+      },
+    ]
+
+    return this.http.patch(routes.Conflicts(`vtex.pages-graphql/${bucketKey}`), data, {
+      metric: 'vbase-resolve-conflicts',
+    })
+  }
+
+  public getConflicts = async () => {
+    return this.http.get(routes.Conflicts('vtex.pages-graphql/userData'), {
+      metric: 'vbase-get-conflicts',
+    })
+  }
+
   public checkForConflicts = async () => {
     let status: number
     try {
-      const response = await this.http.get(routes.File('vtex.pages-graphql', 'store/content.json'), {
+      const response = await this.http.get(routes.File('vtex.pages-graphql/userData', 'store/content.json'), {
```

Comment:
Should this also be extended to accept a `filePath` parameter? To support the template specific files.

Reply:
not necessary. This function is for resolving the conflict just for one file. It's not even required we keep this logic, but I decided to let it to avoid side effects